docs/source/conditions.rst

```diff
@@ -373,6 +387,16 @@ something like:
        - check-success=build: Windows
        - check-success=build: Linux
 
+Alternatively, you may use the following:
+
+.. code-block:: yaml
+
+     conditions:
+       - "#check-failure=0"
+       - "#check-neutral=0"
```

Comment:
This is not an alternative, as explained above and below this kind of condition should not be used.

`- #check-XXX=0` is always true just after a pull request is open or synchronized (force-push or new commits)

All required checks must be provided.